### PR TITLE
Error messages in Responses should be an array of string instead of a single string

### PR DESCRIPTION
**Pull request by luchesar**

### CODE REVIEW

File: api-reference/openapi.yml [hunk 1]
```diff
@@ -933,41 +933,41 @@ components:
...
+          example: ["This user already exists. [name] and [email] must be a unique combination"]
     NotFound:
       type: object
       properties:
-        error:
+        errors:
           type: string
-          description: A msesage describing the not found error
-          example: The resource with id [8be3bb60-5ab4-4bd2-90e6-3b015691a827] does not exist
+          description: A array of msesages describing the not found errors
```
Comment (arun-singh) [inline api-reference/openapi.yml:969]: Type in messages

File: api-reference/openapi.yml [hunk 2]
```diff
@@ -933,41 +933,41 @@ components:
...
     Conflict:
       type: object
       properties:
-        error:
-          type: string
-          description: A msesage describing the conflict error
-          example: This user already exists. [name] and [email] must be a unique combination
+        errors:
+          type: array
+          description: An array of msesages describing the conflict errors
```
Comment (arun-singh) [inline api-reference/openapi.yml:962]: Typo here too

File: api-reference/openapi.yml [hunk 3]
```diff
@@ -933,41 +933,41 @@ components:
...
     UnauthorizedBarer:
       type: object
       properties:
-        error:
-          type: string
-          description: A message describing the error
-          example: "Not Authorization: Bearer <token> provided or wrong value"
+        errors:
+          type: array
+          description: An array messages describing the errors
```
Comment (arun-singh) [inline api-reference/openapi.yml:955]: array **of**

**Review by arun-singh — CHANGES_REQUESTED** [review]

File: api-reference/openapi.yml (hunk 1 above)
Comment (luchesar) [reply api-reference/openapi.yml]: fixed

**Pull request merged**